docs/api/menu-item.md

```diff
@@ -28,7 +28,7 @@ See [`Menu`](menu.md) for examples.
   * `visible` Boolean (optional) - If false, the menu item will be entirely hidden.
   * `checked` Boolean (optional) - Should only be specified for `checkbox` or `radio` type
     menu items.
-  * `registerAccelerator` Boolean (optional) - If false, the accelerator won't be registered
+  * `registerAccelerator` Boolean (optional) _Linux_ _Windows_ - If false, the accelerator won't be registered
```

Comment:
Platform specificity should go at the end of the description:
```
* `registerAccelerator` Boolean (optional) - If false, the accelerator won't be registered with the system, but it will still be displayed. Defaults to `true`. _Linux_ _Windows_
```

Reply:
I copied the style from [here](https://github.com/electron/electron/blob/master/docs/api/app.md#appsetasdefaultprotocolclientprotocol-path-args)

Reply:
Rip cool it seems we apparently have both

In that case np, carry on 😆